packages/solo/test/test-home.js

```diff
@@ -5,9 +5,9 @@ import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
 import bundleSourceAmbient from '@endo/bundle-source';
 import { AmountMath } from '@agoric/ertp';
 import { TimeMath } from '@agoric/time';
-import { Stable } from '@agoric/vats/src/tokens.js';
 import { Far } from '@endo/marshal';
 import { resolve as importMetaResolve } from 'import-meta-resolve';
+import { Stable } from '@agoric/inter-protocol/src/tokens.js';
```

Comment:
This seems suspect to me.  `ag-solo` should be independent of the economy, except for the fact that it may import the sim-chain from `@agoric/cosmic-swingset`.  Can you at least leave a comment behind describing how we would fix this layering?

One idea would be to move the part of the test relying on `Stable` to another package.

Reply:
Glad to hear. This test was the only thing in `@agoric/solo` depending on inter-protocol. Since it's a test I've copied the config over. If the config changes the test should fail.